can_bus/quadruna/FSM/FSM_alerts.json

```diff
@@ -77,10 +85,12 @@
             "description": "Right suspension sensor voltage out of acceptable range (OCSC)."
         },
         "LoadCell1OCSC": {
+            "disabled": true,
```

Comment:
We only allow 21 faults/warnings per board, and so have to disable these to meet that threshold. @Jiajun-Huang is working on disabling them anyway so this should be fine